examples/distribution-openshift/rendered_manifests/configmap-agent.yaml

```diff
@@ -214,8 +214,11 @@ data:
             config:
               extraDimensions:
                 metric_source: kubernetes-proxy
-              port: 29101
+              port: 9101
+              skipVerify: true
               type: kubernetes-proxy
+              useHTTPS: true
+              useServiceAccount: true
             rule: type == "pod" && labels["app"] == "sdn"
```

Comment:
Do we need to update the rule to include the port or we are sure that both `29101` and `9101` are always exposed?

Reply:
This should be possible with OpenShift. Haven't been successful validating this works yet, still looking into it.

Reply:
Done